### PR TITLE
CBA support for iOS

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
This PR adds the ability to build the app with CBA support

### CODE REVIEW

File: app/fetch_preconfig.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            return {
+                response,
+                headers: headersMap,
+                data,
+            };
+        }
+
+        const msg = data.message || '';
+
+        if (this.logToConsole) {
```
Comment (saturninoabril) [inline app/fetch_preconfig.js]: I don't see `logToConsole` declare elsewhere.

File: app/screens/select_server/index.js [hunk 2]
```diff
@@ -33,9 +35,11 @@ function mapStateToProps(state) {
 function mapDispatchToProps(dispatch) {
     return {
         actions: bindActionCreators({
+            ...LoginActions,
```
Comment (saturninoabril) [inline app/screens/select_server/index.js]: 0/5 but would like to have this explicitly pass the necessary actions.

File: app/screens/select_server/select_server.js [hunk 3]
```diff
@@ -212,7 +274,20 @@ export default class SelectServer extends PureComponent {
             return;
         }
 
-        this.pingServer(url);
+        if (LocalConfig.ExperimentalClientSideCertEnable && Platform.OS === 'ios') {
```
Comment (saturninoabril) [inline app/screens/select_server/select_server.js]: Do we need to check this if true? Like:
``LocalConfig.ExperimentalClientSideCertEnable === 'true'``

Comment (enahum) [reply app/screens/select_server/select_server.js]: this is actually a boolean `true`|`false` not a string, the local config is set in the assets config.json file

**Comment by enahum** [comment]
@saturninoabril changes made

File: app/screens/select_server/select_server.js (hunk 3 above)
Comment (saturninoabril) [reply app/screens/select_server/select_server.js]: Ah, right, I got it mixed up with `config.ExperimentalClientSideCertEnable`

**Pull request merged**